src/Components/Web.Extensions/src/InputFile/RemoteBrowserFileStreamOptions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Components.Web.Extensions
+{
+    /// <summary>
+    /// Repesents configurable options for <see cref="RemoteBrowserFileStream"/>.
+    /// </summary>
+    public class RemoteBrowserFileStreamOptions
```

Comment:
Can we mark this as not available in browser-wasm using the UnsupportedOSPlatform attribute? Feel free to do this in a follow up PR